### PR TITLE
SPCTR-3014: Fix - Separator mobile width issue in frontend

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

[Separator] Separator width if set 20-30 in the tablet and if adding any width in the mobile view, the separator will not show up.
https://brainstormforce.atlassian.net/browse/SPCTR-3014

### Screenshots

https://d.pr/v/08j68H

### How has this been tested?

- Add a Separator block on the new page.
- 
- Set the width of the separator block to 80% on the device.
- 
- Go to tablet view, and change the width to 20% - 30%.
- 
- Now change to mobile view, and update any random width of more than 50%.
- 
- Inspect the separator and see the width value.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]
@rajmohanreddy774 Could you please make a Habits of filling out other details in the PR like Screenshot, Type Of Changes, How has this been tested and  Checklist?

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Separator - Resolved an issue in which separator mobile width is not working as intended in frontend.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Separator - Resolved an issue in which the separator mobile width would not work as intended in the frontend.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, only Changelog Update needed.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: done

**Review by adi3890 — APPROVED** [review]

**Pull request merged**